core/domain/stats_jobs_one_off.py

```diff
@@ -150,8 +154,8 @@ def reduce(exp_id, stringified_values):
 
         try:
             exploration = exp_services.get_exploration_by_id(exp_id)
+        # Exploration does not exist.
```

Comment:
Nit: put this comment one line below, and indent it by 4.

Reply:
Done.